File: crawl/pwandora/models/content_moderation.js
const Sequelize = require('sequelize');
module.exports = function(sequelize, DataTypes) {
  return sequelize.define('content_moderation', {
    pwa_id: {
      type: DataTypes.INTEGER,
      allowNull: false,
      primaryKey: true,
      references: {
        model: 'pwa',
        key: 'id'
      }
    },
    child_endangerment: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    inappropriate_content: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    financial_service: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    real_money_gambling: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    illegal_activity: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    health_content_service: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    blockchain_based_content: {
      type: DataTypes.INTEGER,
      allowNull: true
    },
    ai_generated_content: {
      type: DataTypes.INTEGER,
      allowNull: true
    }
  }, {
    sequelize,
    tableName: 'content_moderation',
    timestamps: false,
    indexes: [
      {
        name: "PRIMARY",
        unique: true,
        using: "BTREE",
        fields: [
          { name: "pwa_id" },
        ]
      },
    ]
  });
};

File: be/pwandora/src/main/java/com/ssafy/pwandora/acceptance/entity/ContentModeration.java
package com.ssafy.pwandora.acceptance.entity;

import com.ssafy.pwandora.pwa.entity.Pwa;
import jakarta.persistence.*;
import lombok.Getter;
import lombok.Setter;

@Entity
@Table(name = "content_moderation")
@Getter
@Setter
public class ContentModeration {

    @Id
    @Column(name = "pwa_id")
    private Integer pwaId;

    @OneToOne(fetch = FetchType.LAZY)
    @MapsId
    @JoinColumn(name = "pwa_id")
    private Pwa pwa;

    @Column(name = "child_endangerment")
    private Integer childEndangerment;

    @Column(name = "inappropriate_content")
    private Integer inappropriateContent;

    @Column(name = "financial_service")
    private Integer financialService;

    @Column(name = "real_money_gambling")
    private Integer realMoneyGambling;

    @Column(name = "illegal_activity")
    private Integer illegalActivity;

    @Column(name = "health_content_service")
    private Integer healthContentService;

    @Column(name = "blockchain_based_content")
    private Integer blockchainBasedContent;

    @Column(name = "ai_generated_content")
    private Integer aiGeneratedContent;
}